hw9/task2.js

```diff
@@ -0,0 +1,13 @@
+const book = {
+    title: "JS Basics: I hope i can handle it",
+    author: "Narine Il",
+    year: 1954
+
+}
+
+book.author = "John Smith"
+book.title = "There is always hope"
```

Comment:
you should create new variables by destructuring book object instead of reassigning values on book object